serverless-al2-example/README.md

```diff
@@ -0,0 +1,30 @@
+# Serverless + Golang + Amazon Linux 2 (provided.al2) + CodeSee example
+
+Using AL2 runtime + Datadog serverless plugin + CodeSee Layer Configured
+
+## Installation:
+
+```
+serverless plugin install --name serverless-plugin-datadog
+serverless plugin install --name serverless-go-plugin
+go get github.com/DataDog/datadog-lambda-go
+```
+
+## Set required environment variables for CodeSee:
+
+```
+DD_APM_DD_URL: http://127.0.0.1:8080
+DD_APM_NON_LOCAL_TRAFFIC: true
+DD_APM_ENABLED: true
+CODESEE_BRIDGE_FORWARD_HOST: https://in-datadog.codesee.io
+CODESEE_BRIDGE_TOKEN: <YOUR_CODESEE_TOKEN>
+```
```

Comment:
Can you add a little more information here?

Should these be set in certain files of the example? Or set via some other method? I feel like I've seen several of these in the code. If they are already set in the code, why are we also listing them here?

Reply:
I think you're telling the user to make sure these are set properly in the serverless.yml, is that right?

Reply:
yea I'll add the note: Make sure the following environment variables are set in your `serverless.yml` file.